plugins/include/sqlx.inc

```diff
@@ -127,149 +153,231 @@ native SQL_QuoteStringFmt(Handle:db, buffer[], buflen, const fmt[], any:...);
  *  made for each query to simplify driver support.
  * The handler should look like:
  *
- * @param failstate - One of the three TQUERY_ defines.
- * @param query - Handle to the query, do not free it.
- * @param error - An error message, if any.
- * @param errnum - An error code, if any.
- * @param data - Data array you passed in.
- * @param size - Size of the data array you passed in.
- * @param queuetime - Amount of gametime that passed while the query was resolving.
+ *  failstate - One of the three TQUERY_ defines.
+ *  query - Handle to the query, do not free it.
+ *  error - An error message, if any.
+ *  errnum - An error code, if any.
+ *  data - Data array you passed in.
+ *  size - Size of the data array you passed in.
+ *  queuetime - Amount of gametime that passed while the query was resolving.
  * 
  * public QueryHandler(failstate, Handle:query, error[], errnum, data[], size, Float:queuetime)
+ * 
+ * @note The handle you pass in is a DB tuple, NOT and active connection!
+ * @note The handle does no need to be freed.
+ * @note This function is not guaranteed to be in another thread (in fact - it's not).
+ * 		 You're seeing data "after the fact" and as such to execute another query
+ * 		 you should run SQL_ThreadQuery() again with new data.
  *
- * Note! The handle you pass in is a DB Tuple, NOT an active connection!
- * Note! The handle does not need to be freed.
- * Also note: This function is not guaranteed to be in another thread
- *  (in fact - it's not).  You're seeing data "after the fact", 
- *  and as such to execute another query you should run 
- *  SQL_ThreadQuery again with new data.
+ * @param db_tuple		A tuple handle, returned from SQL_MakeDbTuple().
+ * @param handler		A function to be called when the query finishes. It has to be public.
+ * @param query			The query string.
+ * @param data			Additional data array that will be passed to the handler function.
+ * @param dataSize		The size of the additional data array. 
  */
 native SQL_ThreadQuery(Handle:db_tuple, const handler[], const query[], const data[]="", dataSize=0);
 
 
 /**
- * Executes a query.
- * Returns 1 if the query succeeded.
- * Returns 0 if the query failed.
- * NOTE: You can call this multiple times as long as its parent
- *  connection is kept open.  Each time the result set will be freed
- *  from the previous call.
+ * Executes an already prepared query.
+ * 
+ * @note You can call this multiple times as long as its parent connection kept open. 
```

Comment:
Missing a word here. Also (I think?) the word order in the second sentence should be changed `Each timnne the result set from the previous call will be freed`.